src/playlist.js

```diff
@@ -470,20 +470,6 @@ export const isAes = function(media) {
   return false;
 };
 
-/**
- * Returns whether the current playlist contains fMP4
- *
- * @return {Boolean} true if the playlist contains fMP4
- */
-export const isFmp4 = function(media) {
```

Comment:
We now use `util/codecs.js#isLikelyFmp4Data()` to determine if a segment is fmp4 during  media segment requests. Then we add `segment.isFmp4` to the segment object.